compiler/rustc_ast_lowering/src/delegation.rs

```diff
@@ -258,6 +305,11 @@ impl<'hir> LoweringContext<'_, 'hir> {
         })
     }
 
+    fn get_resolution_id(&mut self, node_id: NodeId, span: Span) -> Result<DefId, ErrorGuaranteed> {
```

Comment:
Also inline and remove, many small functions with similar names make it hard to understand what's going on.